src/composables/useWeb3.ts

```diff
@@ -62,10 +62,17 @@ export function useWeb3() {
       console.log('Provider', auth.provider.value);
       let network, accounts;
       try {
-        [network, accounts] = await Promise.all([
-          auth.web3.getNetwork(),
-          auth.web3.listAccounts()
-        ]);
+        const connector = localStorage.getItem('_lock.connector');
```

Comment:
Is there a more elegant way to detect gnosis? Would be nice if we dont rely on localStorage for this. Maybe you can do similar than Trezor? https://github.com/snapshot-labs/snapshot/pull/1102/files#diff-2042149fed8eaeb1befb9e752306605a4f9a8717caf58211066996ead70638b2R23